MapboxCoreNavigation/NavigationRouteOptions.swift

```diff
@@ -25,7 +25,11 @@ open class NavigationRouteOptions: RouteOptions {
         shapeFormat = .polyline6
         includesSteps = true
         routeShapeResolution = .full
-        attributeOptions = [.congestionLevel, .expectedTravelTime, .maximumSpeedLimit]
+        if profileIdentifier == .walking {
```

Comment:
`profileIdentifier` is mutable, so technically it’s possible for the developer to change the profile identifier to `walking` after initializing the object. We could address that case by conditionally adding `maximumSpeedLimit` in an override of the `URLQueryItems` property instead of this initializer. But maybe it isn’t a big deal.